src/components/keyboard-aware-flat-list.android.js

```diff
@@ -2,7 +2,8 @@
 * @format
 * @flow
 */
-import { FlatList, KeyboardAvoidingView } from 'react-native';
+import { FlatList } from 'react-native';
+import KeyboardAvoidingView from '../components/keyboard-avoiding-view';
```

Comment:
to address https://github.com/wordpress-mobile/gutenberg-mobile/pull/387#discussion_r242525865